### PR TITLE
Updated upgrade to 5.2.1 document

### PR DESCRIPTION
**Pull request by tst-nfarah**
Updated upgrade doc to reflect latest changes: support of postgreSQL databases using RPM for RedHat and Centos.

### CODE REVIEW

File: earth_enterprise/Upgrade.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
 
-Upgrading to GEE 5.2.1 is supported from Earth Enterprise 5.1.3 and 5.2.0 versions.
+Upgrading to GEE 5.2.1 is supported from Earth Enterprise 5.2.0 version using the following steps:
 
-## Upgrading GEE 5.2.0 to 5.2.1
+1. Do NOT uninstall GEE 5.2.0. We recommend that you upgrade GEE 5.2.0 by simply installing GEE 5.2.1. Installing GEE 5.2.1 on top of GEE 5.2.0 will ensure that your postgreSQL databases are backed up and upgraded correctly to the new postgreSQL version used by GEE 5.2.1. [Install Fusion or Earth Server](https://github.com/google/earthenterprise/wiki/Install-Fusion-or-Earth-Server)
 
-1. Do not uninstall GEE 5.2.0. We recommend that you upgrade GEE 5.2.0 by simply installing GEE 5.2.1. Installing GEE 5.2.1 on top of GEE 5.2.0 will ensure that your postgreSQL databases are backed up and upgraded correctly to the new postgreSQL version used by GEE 5.2.1. [Install Fusion or Earth Server](https://github.com/google/earthenterprise/wiki/Install-Fusion-or-Earth-Server)
+Alternatively on Redhat and Centos, you can use upgrade GEE 5.2.0 to GEE 5.2.1 using [RPM] (https://github.com/google/earthenterprise/blob/master/earth_enterprise/BUILD_RPMS.md)
```
Comment (tst-ppenev) [inline earth_enterprise/Upgrade.md]: Language: 'you can use upgrade' -> 'you can upgrade'

File: earth_enterprise/Upgrade.md [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
 
 1. If you decide that you want to uninstall GEE 5.2.0 before installing GEE 5.2.1, first make sure to backup your postgreSQL databases. Please keep in mind that those backup, made by 5.2.0, would not be compatible with GEE 5.2.1 postgreSQL databases.
```
Comment (tst-ppenev) [inline earth_enterprise/Upgrade.md]: Language: 'those backup, made by 5.2.0, would not be compatible' -> 'the database backup, made by 5.2.0, would not be compatible'.

Comment (tst-ppenev) [reply earth_enterprise/Upgrade.md]: Style: Capitalize Postgre SQL.

File: earth_enterprise/Upgrade.md [hunk 3]
```diff
@@ -1,10 +1,10 @@
-# Upgrading Earth Enterprise Fusion and Server to 5.2.1
+# Upgrading Earth Enterprise Fusion and Server to version 5.2.1
 
-Upgrading to GEE 5.2.1 is supported from Earth Enterprise 5.1.3 and 5.2.0 versions.
+Upgrading to GEE 5.2.1 is supported from Earth Enterprise 5.2.0 version using the following steps:
 
-## Upgrading GEE 5.2.0 to 5.2.1
+1. Do NOT uninstall GEE 5.2.0. We recommend that you upgrade GEE 5.2.0 by simply installing GEE 5.2.1. Installing GEE 5.2.1 on top of GEE 5.2.0 will ensure that your postgreSQL databases are backed up and upgraded correctly to the new postgreSQL version used by GEE 5.2.1. [Install Fusion or Earth Server](https://github.com/google/earthenterprise/wiki/Install-Fusion-or-Earth-Server)
```
Comment (tst-ppenev) [inline earth_enterprise/Upgrade.md]: Suggestion: Include commands that user executes.  This does not make it clear whether you're supposed to be running install scripts, or installing packages, and what to do, if you had installed with install scripts, and now you want to install from packages.

Comment (tst-nfarah) [reply earth_enterprise/Upgrade.md]: You mean include:
```cd earth_enterprise/src/installer
sudo ./install_fusion.sh ```

And
``` cd earth_enterprise/src/installer
sudo ./install_server.sh ```

?

If yes, those commands are outlined in the linked document: https://github.com/google/earthenterprise/wiki/Install-Fusion-or-Earth-Server

File: earth_enterprise/Upgrade.md (hunk 1 above)
Comment (tst-nfarah) [reply earth_enterprise/Upgrade.md]: ok

File: earth_enterprise/Upgrade.md [hunk 4]
```diff
@@ -1,14 +1,19 @@
...
+# Upgrading Earth Enterprise Fusion and Server to version 5.2.1
 
-Upgrading to GEE 5.2.1 is supported from Earth Enterprise 5.1.3 and 5.2.0 versions.
+Upgrading to GEE 5.2.1 is supported from Earth Enterprise 5.2.0 version using the following steps:
 
-## Upgrading GEE 5.2.0 to 5.2.1
+1. Do NOT uninstall GEE 5.2.0. We recommend that you upgrade GEE 5.2.0 by simply installing GEE 5.2.1. Installing GEE 5.2.1 on top of GEE 5.2.0 will ensure that your PostgreSQL databases are backed up and upgraded correctly to the new PostgreSQL version used by GEE 5.2.1. [Install Fusion or Earth Server](https://github.com/google/earthenterprise/wiki/Install-Fusion-or-Earth-Server). This can be achieved by running the scripts to install fusion and earth server:
 
-1. Do not uninstall GEE 5.2.0. We recommend that you upgrade GEE 5.2.0 by simply installing GEE 5.2.1. Installing GEE 5.2.1 on top of GEE 5.2.0 will ensure that your postgreSQL databases are backed up and upgraded correctly to the new postgreSQL version used by GEE 5.2.1. [Install Fusion or Earth Server](https://github.com/google/earthenterprise/wiki/Install-Fusion-or-Earth-Server)
+cd earth_enterprise/src/installer
```
Comment (tst-ppenev) [inline earth_enterprise/Upgrade.md]: Style: You could put these commands in a code block. (Add a line of ` ```bash ` before, and a line of ` ``` ` after.)

File: earth_enterprise/Upgrade.md [hunk 5]
```diff
@@ -1,14 +1,19 @@
...
+sudo ./install_fusion.sh 
+sudo ./install_server.sh
 
-1. If you decide that you want to uninstall GEE 5.2.0 before installing GEE 5.2.1, first make sure to backup your postgreSQL databases. Please keep in mind that those backup, made by 5.2.0, would not be compatible with GEE 5.2.1 postgreSQL databases.
+
+Alternatively of using scripts, on Redhat and Centos, you can upgrade GEE 5.2.0 to GEE 5.2.1 using [RPM] (https://github.com/google/earthenterprise/blob/master/earth_enterprise/BUILD_RPMS.md)
```
Comment (tst-ppenev) [inline earth_enterprise/Upgrade.md]: Minor language: I think changing 'using RPM' to 'using RPM packages' would complete the sentence.

File: earth_enterprise/Upgrade.md [hunk 6]
```diff
@@ -1,14 +1,19 @@
...
+
+1. If you decide that you want to uninstall GEE 5.2.0 before installing GEE 5.2.1, first make sure to backup your PostgreSQL databases. Please keep in mind that the database backup, made by 5.2.0, would not be compatible with GEE 5.2.1 PostgreSQL databases.
 
     * Create a backup folder: mkdir -p /tmp/MyBackupFolder
```
Comment (tst-ppenev) [inline earth_enterprise/Upgrade.md]: Style: You could make the commands monospaced by putting them inside '`' marks.

Comment (tst-nfarah) [reply earth_enterprise/Upgrade.md]: ok

File: earth_enterprise/Upgrade.md (hunk 5 above)
Comment (tst-nfarah) [reply earth_enterprise/Upgrade.md]: sounds good

File: earth_enterprise/Upgrade.md (hunk 4 above)
Comment (tst-nfarah) [reply earth_enterprise/Upgrade.md]: sure

File: earth_enterprise/Upgrade.md [hunk 7]
```diff
@@ -1,14 +1,20 @@
...
 
-1. Do not uninstall GEE 5.2.0. We recommend that you upgrade GEE 5.2.0 by simply installing GEE 5.2.1. Installing GEE 5.2.1 on top of GEE 5.2.0 will ensure that your postgreSQL databases are backed up and upgraded correctly to the new postgreSQL version used by GEE 5.2.1. [Install Fusion or Earth Server](https://github.com/google/earthenterprise/wiki/Install-Fusion-or-Earth-Server)
+```bash
+cd earth_enterprise/src/installer
+sudo ./install_fusion.sh 
+sudo ./install_server.sh
+```
 
-1. If you decide that you want to uninstall GEE 5.2.0 before installing GEE 5.2.1, first make sure to backup your postgreSQL databases. Please keep in mind that those backup, made by 5.2.0, would not be compatible with GEE 5.2.1 postgreSQL databases.
+Alternatively of using scripts, on Redhat and Centos, you can upgrade GEE 5.2.0 to GEE 5.2.1 using [RPM packages] (https://github.com/google/earthenterprise/blob/master/earth_enterprise/BUILD_RPMS.md)
```
[the remark this answers is not in the document]
Comment (tst-nfarah) [reply earth_enterprise/Upgrade.md]: yes, correct. I'll use one of the proposed phrases.

**Pull request merged**